test/ginkgo-ext/scopes.go

```diff
@@ -538,7 +570,7 @@ func wrapTest(f interface{}) interface{} {
 func calculateCounters(s *scope, focusedOnly bool) (int, bool) {
```

Comment:
I wish I could understand what is meant to happen, but right now I find it very difficult and I do have many too many doubt about what's going.